src/freertos_drivers/ti/CC3200GPIO.hxx

```diff
@@ -171,6 +171,7 @@ public:
         volatile uint8_t *ptr = reinterpret_cast<uint8_t *>(
             GPIO_BASE + (((unsigned)GPIO_PIN) << 2));
         *ptr = value ? 0xff : 0;
+        __asm__ volatile("\t nop\n");
```

Comment:
Can you add a comment as to why this nop is required?

Reply:
done